tests.py

```diff
@@ -319,3 +319,80 @@ def test_properties_columns():
     assert a.columns == [0]
     a.columns = [5, 6]
     assert a.columns == [4, 5]
+
+
+def test_logging_filter_defaults(caplog):
+    logging.disable(logging.NOTSET)
+    logging.getLogger("anonip").setLevel(logging.CRITICAL)
+
+    logger = logging.getLogger("filter_defaults")
+    logger.addFilter(anonip.AnonipFilter())
+    logger.setLevel(logging.INFO)
+
+    logger.info("192.168.100.200 string")
+    logger.info("1.2.3.4 string")
+    logger.info("2001:0db8:85a3:0000:0000:8a2e:0370:7334 string")
+    logger.info("2a00:1450:400a:803::200e string")
+
+    assert caplog.record_tuples == [
+        ("filter_defaults", logging.INFO, "192.168.96.0 string"),
+        ("filter_defaults", logging.INFO, "1.2.0.0 string"),
+        ("filter_defaults", logging.INFO, "2001:db8:85a0:: string"),
+        ("filter_defaults", logging.INFO, "2a00:1450:4000:: string"),
+    ]
+
+    logging.disable(logging.CRITICAL)
```

Comment:
I'd prefer having a pytest fixture for this in order to make sure logging is disabled, even if the test fails.

```
@pytest.fixture
def disable_logging_afterwards():
    yield
    logging.disable(logging.CRITICAL)
```

Same goes for the other tests.

Reply:
I suspect, that the call to `logging.disable` is a remnant of early test implementations. It looks like we could remove that completely (#53). If that is the case, the fixture wouldn't be needed neither.

Reply:
Absolutely right!